### PR TITLE
Update customer-lockbox-requests.md

### PR DESCRIPTION
**Pull request by msbemba**
Correcting the steps to Customer lockbox on or off

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2110

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/compliance/customer-lockbox-requests.md [hunk 1]
```diff
@@ -69,11 +69,9 @@ You can turn on Customer Lockbox controls in the Microsoft 365 admin center. Whe
 
 1. Using a work or school account that has either the global administrator or the **Customer Lockbox access approver** role assigned, go to [https://admin.microsoft.com](https://admin.microsoft.com) and sign in.
 
-2. Choose **Settings > Security & privacy**.
+2. Choose **Settings > Org Settings**.
 
-    ![Edit the Customer Lockbox settings in the admin center](../media/CustomerLockbox2.png)
-
-3. On the **Customer Lockbox** tile, choose **Edit**, and then move the toggle to **On** or **Off** to turn the feature on or off.
+3. Click **Services** and Click on **Customer Lockbox**, Then choose **Edit**, and then move the toggle to **On** or **Off** to turn the feature on or off.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/customer-lockbox-requests.md]:
```suggestion
3. Select **Services** > **Customer Lockbox** > **Edit**, and then move the toggle to **On** or **Off** to turn the feature on or off.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks.

**Pull request merged**